packages/backend/src/api/controllers/tvl/TvlController.ts

```diff
@@ -67,9 +67,11 @@ export class TvlController {
       await Promise.all([
         this.aggregatedReportRepository.getHourly(
           getHourlyMinTimestamp(timestamp),
+          ValueType.TVL,
         ),
         this.aggregatedReportRepository.getSixHourly(
           getSixHourlyMinTimestamp(timestamp),
+          ValueType.TVL,
         ),
         this.aggregatedReportRepository.getDaily(),
```

Comment:
what about ValueType there?

Reply:
Shouldn't be a part of describing the endpoints? For now, we focus on only on aggregates and yet not expose the rest of the logic I presume